packages/angular/projects/ui-angular/src/lib/ui-angular.module.ts

```diff
@@ -16,8 +16,59 @@ import { AmplifyForceNewPasswordComponent } from './components/amplify-force-new
 import { AmplifyFederatedSignInComponent } from './components/amplify-federated-sign-in/amplify-federated-sign-in.component';
 import { AmplifyFederatedSignInButtonComponent } from './components/amplify-federated-sign-in-button/amplify-federated-sign-in-button.component';
 
+import {
+  AmplifyS3Album as S3Album,
+  AmplifyS3ImagePicker as S3ImagePicker,
+  AmplifyS3Text as S3Text,
+  AmplifyS3TextPicker as S3TextPicker,
+  AmplifyS3Image as S3Image,
+  AmplifyPicker as picker,
+  AmplifyPhotoPicker as photoPicker,
+  AmplifyChatbot as chatbot,
+  AmplifyToast as toast,
+  AmplifyInput as input,
+  AmplifyButton as button,
+} from './directives/proxies';
+
+import {
+  AmplifyChatbot,
+  AmplifyButton,
+  AmplifyInput,
+  AmplifyToast,
+  AmplifyS3Album,
+  AmplifyS3ImagePicker,
+  AmplifyS3Text,
+  AmplifyS3TextPicker,
+  AmplifyS3Image,
+  AmplifyPicker,
+  AmplifyPhotoPicker,
+} from '@aws-amplify/ui-components/dist/components';
+
+customElements.define('amplify-button', AmplifyButton);
+customElements.define('amplify-input', AmplifyInput);
+customElements.define('amplify-toast', AmplifyToast);
+customElements.define('amplify-chatbot', AmplifyChatbot);
+customElements.define('amplify-s3-album', AmplifyS3Album);
+customElements.define('amplify-s3-image-picker', AmplifyS3ImagePicker);
+customElements.define('amplify-s3-text', AmplifyS3Text);
+customElements.define('amplify-s3-text-picker', AmplifyS3TextPicker);
+customElements.define('amplify-s3-image', AmplifyS3Image);
+customElements.define('amplify-picker', AmplifyPicker);
+customElements.define('amplify-photo-picker', AmplifyPhotoPicker);
```

Comment:
This is a nice hack! Hopefully this will help with tree shaking too, since these aren't lazy loaded.

Reply:
👍